rest-api-model-server/src/main/kotlin/org/modelix/sample/restapimodelserver/LanguageConfig.kt

```diff
@@ -21,9 +18,7 @@ import javax.enterprise.event.Observes
 class LanguageConfig {
 
     fun configure(@Observes event: StartupEvent) {
-        MPSLanguageRegistry.register(jetbrains_mps_lang_core.INSTANCE)
-        MPSLanguageRegistry.register(University_Schedule.INSTANCE)
-        MPSLanguageRegistry.register(org_modelix_model_repositoryconcepts.INSTANCE)
+        GeneratedLanguages.registerAll()
 
         // Add our custom serializer for the REST representation of nodes
         INodeReferenceSerializer.register(serializer)
```

Comment:
Can't the `INodeSerializer.Companion` be used here now, too?

Reply:
Don't know. Where can I find details on it? Maybe best we add this in a new PR?